magento/magento2
# Fix #27523: throw informative errors in setup:db:generate-patch

### Description (*)
In most cases, if there is indeed a write permission issue, the error does show the complete path to directory which cannot be created:
![Screenshot from 2020-04-04 17-21-06](https://user-images.githubusercontent.com/24894168/78453213-e6f2cd80-7698-11ea-807a-1f858de8cd4c.png)
However, if the requested module *does not exist*, the console command will compile an invalid(incomplete) target path and will show a rather cryptic error message:
![Screenshot from 2020-04-04 17-09-33](https://user-images.githubusercontent.com/24894168/78453241-0b4eaa00-7699-11ea-8745-4e630450314f.png)

### Fixed Issues (if relevant)

1.  Fixes magento/magento2#27523: CLI Patch Generator: Lack of information what path cannot be created

### Manual testing scenarios (*)
1. Run `php bin/magento setup:db-declaration:generate-patch Vendor_Module SomePatch`, while there isn't any registered modules with name matching `Vendor_Module`
2. You should see an informative error message like `Cannot find a registered module with name "Vendor_Module" `

### Contribution checklist (*)
